Src/TestSample.BroadInt.Tests/Users/UserThenSteps.cs

```diff
@@ -0,0 +1,45 @@
+using FluentAssertions;
+using Grpc.Core;
+using TechTalk.SpecFlow;
+using TechTalk.SpecFlow.Assist;
+using TestSample.Domain;
+using TestSample.Domain.Users;
+using TestSample.Grpc.proto;
+
+namespace TestSample.BroadInt.Tests.Users;
+
+[Binding]
+public class UserThenSteps : UserBaseSteps
+{
+    protected UserThenSteps(ScenarioContext scenarioContext, UserGrpcClient client) : base(scenarioContext, client) { }
+
+    [Then(@"the a user is returned with the following properties")]
+    public void ThenTheAUserIsReturnedWithTheFollowingProperties(Table table)
+    {
+        var userResult = ScenarioContext.Get<Result<UserMessage, RpcException>>();
+        userResult.IsSuccess.Should().BeTrue();

```

Comment:
Is this testing for the return code here? 
If not, I think response code based on what adaptor needs to be included in the Acceptance Criteria.